src/chocolatey/infrastructure.app/commands/ChocolateySupportCommand.cs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright © 2017 - 2025 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using chocolatey.infrastructure.app.attributes;
+using chocolatey.infrastructure.app.configuration;
+using chocolatey.infrastructure.commandline;
+using chocolatey.infrastructure.commands;
+using chocolatey.infrastructure.logging;
+
+namespace chocolatey.infrastructure.app.commands
+{
+    [CommandFor("support", "provides support information", Version = "2.5.0")]
+    public class ChocolateySupportCommand : ICommand
+    {
+        public void ConfigureArgumentParser(OptionSet optionSet, ChocolateyConfiguration configuration)
+        {
+            // Intentionally left blank
+        }
+
+        public void ParseAdditionalArguments(IList<string> unparsedArguments, ChocolateyConfiguration configuration)
+        {
+            // Intentionally left blank
+        }
+
+        public void Validate(ChocolateyConfiguration configuration)
+        {
+            // Intentionally left blank
+        }
+
+        public void HelpMessage(ChocolateyConfiguration configuration)
+        {
+            this.Log().Info(ChocolateyLoggers.Important, "Support Command");
+            this.Log().Info(@"
+As an open-source user of Chocolatey CLI, we are not able to
+ provide private support. See https://chocolatey.org/support
+ for details.
+");
+
+        }
+
+        public void DryRun(ChocolateyConfiguration configuration)
+        {
+            Run(configuration);
+        }
+
+        public void Run(ChocolateyConfiguration config)
+        {
+            this.Log().Info(@"
+Unfortunately, we are unable to provide private support for
+ open-source users. However, there are a lot of avenues for open
+ source users within the community to get the help they need.
+
+If you are an open-source user, please visit
+ https://chocolatey.org/support for the full list of options, or
+ https://docs.chocolatey.org for our open source documentation.
+");
+        }
+
+        public bool MayRequireAdminAccess()
+        {
+            return false;
+        }
+
+#pragma warning disable IDE1006
+        [Obsolete("This method or member is obsolete and will be removed in version 7.0 of Chocolatey Licensed Extension.")]

```

Comment:
These messages are incorrect, you'll want to update them to be for CLI.

Reply:
Doh! Too much copy/pasta! Thanks for catching that!